docs/user-guide/data/work-on-data.md

```diff
@@ -25,13 +25,13 @@ Everything about the resource is displayed below the media viewer (when there is
 
 ![Resource toolbar description.](../../assets/images/resource-toolbar.png)
 
-##### Audio file
```

Comment:
for this and the subsequent headers, the level was incremented by 2 instead of 1